### DISCUSSION TITLE
API to delete attachment [Jun 27]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6013] [Discord Threads]
> 🙋‍♂️ **fromq1234**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Not supported currently  
#  
> 🙋‍♂️ **fromq1234**  
#### How can I delete files in my downloads directory without messing everything up then?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1111501285355692067)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
